### PR TITLE
fix_ui

### PR DESCRIPTION
**Pull request by hienbx94**
I have created a pull request. Please check it.

### CODE REVIEW

**Comment by hellovietnam93** [comment]
ok

File: app/assets/stylesheets/courses.css.scss [hunk 1]
```diff
@@ -1,204 +1,87 @@
...
-    div:hover {
-      background: #D6D6D6;
-    }
-  }
-  #subject_button_action {
-    text-align: center;
-    margin-top: 10px 0px;
-  }
+.check_finish {
+  float: right
```
Comment (trantube94) [inline app/assets/stylesheets/courses.css.scss]: thiếu ;

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        <div class="col-md-12 subject_button_action" >
+          <%= f.submit t("tasks.update"), class: "btn btn-primary" %>
+          <%= render "finish_form" %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
+
+  <div class="col-md-3">
+      <ul class="nav nav-tabs">
```
Comment (trantube94) [inline app/views/courses/show.html.erb]: indent

File: app/views/courses/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          <%= t "subjects.start_date" %>
+          <%= @user_subject.created_at.strftime Settings.date_format %>
+        </li>
+        <li><%= t "subjects.end_date" %></li>
+        <li><%= t "subjects.status" %></li>
+      </ul>
+      
+      <h4><%= t "subjects.introduction" %></h4>
+      <div class="description">
+       <%= @user_subject.subject.description %>
```
Comment (trantube94) [inline app/views/courses/show.html.erb]: add space

**Comment by VuongLong** [comment]
ok

**Comment by lannp** [comment]
OK

**Pull request merged**